Documentation/Signoffs/SteeringSystem.md

```diff
@@ -0,0 +1,152 @@
+
+To verify that this ratio would provide accurate mapping of 960 to 360 degrees, we created a MATLAB script to iterate through all possible positions of the encoder as seen in the figure below.
+
+![image](https://user-images.githubusercontent.com/100802413/202862273-5f44f45a-ed0a-4e65-bb63-7dd475245d4e.png)
+
+_Figure 8: MATLAB Verfication for all possible Angles_
+
+##### 6. Gear and Mounting Design
+The size constraints for the next closest item in the car are as follows:
+- Closest point to the left of the mounting plate: 12 cm
+- Closest point behind the mounting plate: 6 cm
+- Closest point below the mointing plate (barrier determined to be for operator's feet): 10 cm
+- Closest point to the right of the steering column: 7 cm
+- Closest point to above the steering column: 9 cm
+
+The steering column has an inch diameter, so the small gear must have a center bore of exactly 1 inch (2.54 cm). To maintain a small size, the small gear was chosen to have a 6 cm pitch diameter to fit around the steering column and comply with size restrictions around the column. When chosing 6 cm for the small gear, 16 cm must be the pitch diameter of the large gear based on the Law of Gearing. The center bore of the large gear must fit the knob of the encoder such that the rotation of the gear is the same rotation for the encoder. In this case, the center bore of the large gear must be 8 mm in diameter and a 0.5 mm notch to secure the knob of the encoder. 
+
+The mounting bracket of the encoder will be connect to the frame of the Leaf via an adapter plate. The gear connected to the encoder needs to be on the same plane as the gear from the steering column to allow the gears to mesh properly. This was done by creating the adapter plate to allow the mounting bracket to be rotated 45 degrees from the frame and be parallel to the steering column. Then the depth of the adapter was determined to be 12.7063 mm based off of the space needed to have the gears far enough to mesh, but also have the gears as close as possible to reduce the backlash between the teeth. These gears will be 3D printed in the Maker's Space in the Angelo & Jennette Volpe Library.
+
+The furthest point that the design extends to the right is the small gear on the steering column, and its farthest point protrudes 2.03 cm from edge of the steering column. The furthest point to the left is the large gear and it should extend 10.38 cm past the edge of the mounting plate. The model should not extend below the mounting plate by any more than 1 cm. The back of the mounting plate is the furthest back that the design goes and should not conflict with the wall. The furthest up the design goes is the 10.45 cm due to the large gear and the closest point is above the steering column. Since the large gear is offset, the closet point to it is 17 cm above the mounting plate. With the specifications of the design, none of the boundaries on the car are exceeded and the size constraints are met.
+
+### BOM
```

Comment:
This design looks good. I'm ready to signoff. But, I am a bit concerned about the encoder selection. It is somewhat expensive. I recommend looking for a lower cost option (either a lower cost encoder or maybe a rotary potentiometer). However, I'll leave that as a decision for the team. Keep in mind that the total budget is 1000 dollars without specifically showing a unique need for additional funds. So, be careful to consider if the money may be needed somewhere else in the project.